charts/posthog/templates/_snippet-plugins-deployment.tpl

```diff
@@ -37,31 +37,47 @@ spec:
         {{- end }}
     spec:
       serviceAccountName: {{ template "posthog.serviceAccountName" .root }}
+
       {{- if .params.affinity }}
       affinity:
         {{- toYaml .params.affinity | nindent 8 }}
       {{- end }}
+
       {{- if .params.nodeSelector }}
       nodeSelector:
         {{- toYaml .params.nodeSelector | nindent 8 }}
       {{- end }}
+
       {{- if .params.tolerations }}
       tolerations:
         {{- toYaml .params.tolerations | nindent 8 }}
       {{- end }}
+
       {{- if .params.schedulerName }}
       schedulerName: "{{ .params.schedulerName }}"
       {{- end }}
+
       {{- if .params.priorityClassName }}
       priorityClassName: "{{ .params.priorityClassName }}"
       {{- end }}
+
       {{- if .root.Values.image.imagePullSecrets }}
       imagePullSecrets:
         {{- toYaml .root.Values.image.imagePullSecrets | nindent 8 }}
       {{- end }}
+
+      # I do not know for sure if the old one has been used anywhere, so do both :(
+      {{- if .root.Values.image.pullSecrets }}
```

Comment:
Should this be an if else with the above definition?

What's the `imagePullSecrets` / `pullSecrets` distinction?

Reply:
`imagePullSecrets` is something that's been here for ages, but not documented and idk if it has been used.

`pullSecrets` is what has been added now - `image.pullSecrets` is imo nicer than `image.imagePullSecrets`, and naming is inline with our other `image.*` things. I'd like to deprecate/remove `imagePullSecrets` eventually

It could probably be an else, however